packages/mongo/collection.js

```diff
@@ -388,8 +389,7 @@ Mongo.Collection._rewriteSelector = function (selector) {
       // override the ones set on $regex.
       if (value.$options !== undefined)
         ret[key].$options = value.$options;
-    }
-    else if (_.contains(['$or','$and','$nor'], key)) {
+    } else if (_.contains(['$or','$and','$nor'], key)) {
```

Comment:
This isn't super relevant, but how do we feel about rewriting code like this as:

```js
{ $or: 1, $and: 1, $nor: 1 }[key]
```